src/meshapi/views/forms.py

```diff
@@ -160,8 +160,8 @@ def process_join_form(r: JoinFormRequest, request: Optional[Request] = None) ->
         logging.debug(r.street_address, r.city, r.state, r.zip_code)
         return Response(
             {
-                "detail": "Non-NYC registrations are not supported at this time. Check back later, "
-                "or send an email to support@nycmesh.net"
+                "detail": "Non-NYC registrations are not supported at this time. Please double check your zip code, "
+                "try again later, or send an email to support@nycmesh.net"
```

Comment:
```suggestion
                "detail": "Non-NYC registrations are not supported at this time. Please double check your zip code, "
                "or send an email to support@nycmesh.net"
```

I don't think "try again 'later'" is useful instruction. Let's just remove that.

Reply:
(I think that one was my fault lol)